libs/go/zmscli/import.go

```diff
@@ -234,6 +237,9 @@ func (cli Zms) importRolesOld(dn string, lstRoles []interface{}, validatedAdmins
 					}
 				}
 				_, err = cli.AddRoleMembers(dn, rn, roleMembers)
+				if err != nil {
```

Comment:
this check here is not correct. we already handle the error on line 253 where we check if we should report errors or not.

Reply:
This `err` is declared inside the `if` block when `:=` is used, and is lost before the check on 253.

https://github.com/AthenZ/athenz/blob/47a7734a3b11e1d8f425602b1bc9e6fb6035e096/libs/go/zmscli/import.go#L222